packages/default/scss/colorpicker/_layout.scss

```diff
@@ -392,12 +392,10 @@
             margin-top: ($colorpicker-margin * 2);
             text-align: center;
         }
-    }
 
-    .k-rtl,
-    [dir="rtl"] {
-        .k-flatcolorpicker {
 
+        .k-rtl &,
+        &[dir="rtl"] {
```

Comment:
The `dir` attribute can also be set at the root of the page, not directly to the element with class `k-flatcolorpicker`. So, I suggest adding a third selector here:
```
.k-rtl &,
[dir="rtl"]&,
[dir="rtl"] & { ... }
```

For reference, check badge styles: https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/badge/_layout.scss#L157